src/action/mod.rs

```diff
@@ -344,6 +344,14 @@ pub enum ActionError {
     /// A MacOS (Darwin) plist related error
     #[error(transparent)]
     Plist(#[from] plist::Error),
+    #[error(transparent)]
+    ParseNixConfig(#[from] nix_config_parser::ParseError),
+    #[error("Could not merge Nix configuration for keys {}; consider removing them from {1} or removing your existing configuration with `rm {1}`", .0
```

Comment:
I think they cannot remove individual lines with `rm` we should advise using their editor.